server/index.js

```diff
@@ -1,12 +1,113 @@
 const express = require('express');
+const axios = require('axios');
+const cors = require('cors');
 const app = express();
 const port = 3000;
 
 app.use(express.json());
+app.use(cors());
 app.use(express.urlencoded({ extended: false }));
 app.use('/items/:itemId', express.static('client'));
 
 app.listen(port, () => {
```

Comment:
Express executes in order, the app.listen() should be at the bottom of the file.